.vscode/settings.json

```diff
@@ -1,7 +1,12 @@
 {
-    "python.linting.pylintArgs": ["--load-plugins=pylint_django"],
     "python.linting.enabled": true,
-    "python.linting.pylintEnabled": false,
+    "python.linting.pylintEnabled": true,
+    "python.linting.pylintArgs": [
+        "--disable=C0111", // missing docstring
```

Comment:
Does VSCode support comments in its config file JSON?

Reply:
It actually does.

https://code.visualstudio.com/docs/languages/json#_json-with-comments

Normal JSON this would be a problem in.

I believe it's more [json5](https://json5.org/) style.